exporter/local/export.go

```diff
@@ -101,6 +101,9 @@ func (e *localExporterInstance) Export(ctx context.Context, inp *exporter.Source
 	}
 
 	isMap := len(inp.Refs) > 0
+	if len(inp.Attestations) > 0 {
```

Comment:
I do not understand this. Needs a comment. I still don't understand it even after reading the commit message (it says that there is a "flaw" and now things work because of a "quirk" but not what this line really does).